src/js/webdriver.js

```diff
@@ -83,9 +87,9 @@ gpii.webdriver.execute = function (that, fnName, eventName, fnArgs) {
  *
  * A helper function to assist in navigating from a single Fluid IoC test sequence step.  See the docs for details.
  *
- * @param that - The component itself
- * @param args {Array} - An array representing a series of function names and arguments.  Each array's first element is a function name. The remaining arguments are passed to the function.
- * @returns a Promise that will be resolved when navigation is complete, or rejected if there is an error.
+ * @param {Object} that - The component itself
```

Comment:
For all "the component itself" please add the grade name rather than {Object}

Reply:
Done, throughout.